src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsSelector.js

```diff
@@ -244,7 +244,8 @@ function MoneyRequestParticipantsSelector({
             false,
             {},
             [],
-            true,
+            // We don't want the user to be able to invite individuals when they are in the "Distance request" flow for now. This functionality is being built in #23291.
```

Comment:
I appreciate this clear and informative comment!

Nit: Maybe we could, in a new line, write...

> This functionality is being built here: (fulll issue URL)